### ISSUE TITLE
[question] Curious about the every minute refreshes

### ISSUE THREAD

**Issue by plsnotracking** [ahertel/wholefoods-delivery-slot-finder#4]
I'm surprised that Amazon website didn't block us after so many refreshes and doesn't think we are bot. I went through the script and couldn't see any sort of cookie magic. Just curious on any ideas that you might have about how that works. Thank you for this, it's a life saver. Stay safe.

**Comment by ahertel** [comment]
Hi @plsnotracking. Sorry for the late response. Yes, I think it's a serious concern that Amazon might flag our computers/networks/accounts for using this script. At the moment my neighbor, and friend in a nearby town and I have all been running this script for the last 24+ hours without a single real slot appearing (beside two slots that vanished when clicked). Perhaps this is a sign of our accounts being flagged/blocked, or perhaps it's just a lack of slots. For the last few hours I've been running the script with a VPN and I'm going to leave it overnight.

Have you run into any long periods of times without slots or any signs that Amazon might be blocking us? Looking forward to continuing this discussion.

**Comment by iblind** [comment]
@ahertel please let us know whether the VPN has helped or not, v. curious about whether refreshing from the same client is problematic. IMO, refreshing every sixty seconds from the same browser should be normal user behavior — it strikes me that this is precisely what a user may do (and likely at a higher frequency) when they can't find a slot.

**Comment by changethisusername** [comment]
I ran into the same issue yesterday. The script would catch a slot and then as soon as I click it they are gone. This happened about 5 times over a span of about 8h, I was finally able to get a slot in the afternoon, I was using a VPN as well. I also cut the reload time from ~60s to about 38s, which worked great.

**Comment by litingfeng** [comment]
I started to use this script yesterday and successfully placed my order.

But today, I saw time-slot like 7 times but was unable to place the order, after click 'placeorder', the page jumped to 'reserve a time slot'. BYW, I turned on vpn.

Anybody have suggestions?

I am 90% that I've been blocked out.

**Comment by litingfeng** [comment]
I tried another computer without vpn, but same wifi, can't place order either.

**Comment by iblind** [comment]
Tried yesterday for the first time, worked after about ~3hrs. No VPN. 

For VPN users, curious as to the reasoning for taking that route — their servers don't need to track your IP to ID you since you're already logged into your account. If there's some sort of rate limit they'd like to implement, wouldn't it be on an account basis rather than IP basis?

**Comment by changethisusername** [comment]
That’s just problem solving 101. Something doesn’t work they way it did before. First we blame it on caching, then try a VPN, then different browser. These 3 options narrow down where the issue lies most of the time. For example is the issue local server side? Most bans are not connected to accounts but to IPs, hence a different IP can place orders no problem.

**Comment by plsnotracking** [comment]
So what I've gathered is - Your internet needs to be fast and your cart needs to be slim (for Whole Foods). The bigger the number of products the more time it takes to check if this products are available or not. 
Hence, I think a lot of us are seeing slots but when we try to click it, they disappear. I've seen cases where I click on pay to order and then I fail. @ahertel it would be actually better to claim a slot in Amazon Fresh (they've now stopped inviting new people to the programme) since it gives you 59 minutes to add items to the cart rather than the whole foods slot. Feel free to feedback.

I was able to order stuff from Amazon Fresh twice but have only been able to use the script only once to order for Whole Foods. Also everyone, be mindful of booking slots, because others are in the same situation as us, don't book too many :)
